clients/ts/signalr/src/HttpClient.ts

```diff
@@ -163,6 +163,7 @@ export class DefaultHttpClient extends HttpClient {
             xhr.open(request.method, request.url, true);
             xhr.withCredentials = true;
             xhr.setRequestHeader("X-Requested-With", "XMLHttpRequest");
+            xhr.setRequestHeader("Content-Type", "text/plain;charset=UTF-8");
```

Comment:
Pop a comment on here indicating that this is needed by React-Native on Android, so we don't accidentally remove it later.

Reply:
Will do. Do you prefer at end of line or before the line?

Reply:
Before the line please

Reply:
Our TS linter is pretty strict. I think you need to remove the space at the end of the comment

> EXEC : error : /home/travis/build/aspnet/SignalR/clients/ts/signalr/src/HttpClient.ts[166, 96]: trailing whitespace